### DISCUSSION TITLE
How do I clear msgs or notications

### DISCUSSION THREAD

**Discussion by shy00n** [scalameta/coc-metals#365] [Q&A] [answered]
These msgs/notifications are great; however, I would like to clear up. What's the command / keybind? Thanks,

![image](https://user-images.githubusercontent.com/72399256/101736840-08128600-3a92-11eb-9daf-6ac58b81ac0e.png)

**Comment by ckipp01** [comment]
Hello! Ironically I actually _just_ asked the same question on the [coc.nvim Gitter chat](https://gitter.im/neoclide/coc.nvim?at=5fd1d26d1f36292125e121b6). I have no idea how these are meant to be used, and they are from a new update in coc.nvim. Once I hear back and figure out how a user is supposed to interact with these I'll post back in here.

**Reply by ckipp01** [reply]
Also, you should be able to just use
```vim
Plug 'neoclide/coc.nvim', {'branch': 'release'}
```
instead of the master branch, (which I assume you're using), which won't have this behavior yet.

**Comment by ckipp01** [comment, marked as answer]
So to answer your initial question @shy00n, you can use the following to close the floats:

```vim
coc#float#close_all()
```
https://gitter.im/neoclide/coc.nvim?at=5fd1e13a0697c1210db1a48a

**Reply by ckipp01** [reply]
Also you can see some more context here https://github.com/neoclide/coc.nvim/issues/2472.

Feel free to comment in there with any questions about the new floats.

**Comment by shy00n** [comment]
Thanks that helps! maybe I will assign it to a key (suggestions welcomed).
Would be nice to just disappear after a few sec like other editors.
